sym_object.py

```diff
@@ -485,7 +489,7 @@ def resizeUIObject(self, item, force_resize, size):
                         rightmost_object.scenePos().x() - rightmost_object.width
 
         if item.connected_objects:
-            if x_diff < 0:
```

Comment:
I'm not sure if we should be doing `x_diff < size` because the parent tends to resize even though it has enough space inside it (which is why I had changed it to < 0). Thought it might be better to save some space on the canvas but we can address it later on.

Reply:
when it was 0 the parent wouldn't resize correctly with grand or greatgrandchildren

Reply:
I see, thanks for letting me know. I can look into how to make it work for both cases

Reply:
maybe tweaking the way x_diff is calculated might make it work for both cases